From: Werner Koch <wk@gnupg.org>
Date: Wed, 5 Jul 2017 12:08:26 +0200
Subject: syscfg: Add lock-obj-pub file for ia64-unknown-linux-gnu.

* src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h: new.
* src/Makefile.am (lock_obj_pub): Add it.
--

GnuPG-bug-id: 3242
Signed-off-by: Werner Koch <wk@gnupg.org>
(cherry picked from commit 0f5c3f67946fb5389df5b9963e6976b462bf5726)
---
 src/Makefile.am                                  |  1 +
 src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h | 25 ++++++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h

diff --git a/src/Makefile.am b/src/Makefile.am
index 398ec5e..e4ebb6a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -73,6 +73,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.x86_64-pc-linux-gnux32.h        \
         syscfg/lock-obj-pub.x86_64-pc-linux-musl.h          \
 	syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h      \
+	syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h        \
 	syscfg/lock-obj-pub.mingw32.h
 
 
diff --git a/src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h
new file mode 100644
index 0000000..7cf980a
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.ia64-unknown-linux-gnu.h
@@ -0,0 +1,25 @@
+## lock-obj-pub.ia64-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[40];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
